versions.csv

```diff
@@ -1,5 +1,5 @@
 AssemblyPath, Version
-iothub\device\src\Microsoft.Azure.Devices.Client.csproj, 1.41.2
```

Comment:
The nuget push step would have failed in this attempt, right? (attempting to push bits with a version number that already exists)

Reply:
I pushed updated version for Microsoft.Azure.Devices instead of Devices.Client by mistake and it worked as expected

Reply:
We might want to unlist it from nuget if Microsoft.Azure.Devices was pushed out without any code changes. Or else, we might want to add a note to our release notes that this version was pushed out but had no changes. This is just to help with user confusion when they look at the package but don't see any associated release notes.